### PR TITLE
Custom subject in invitation email

### PR DESCRIPTION
**Pull request by meluru**
Closes: #2783

### CODE REVIEW

File: indico/modules/events/registration/forms.py [hunk 1]
```diff
@@ -144,8 +144,9 @@ def pre_validate(self, form):
 
 class InvitationFormBase(IndicoForm):
     _invitation_fields = ('skip_moderation',)
-    _email_fields = ('email_from', 'email_body')
+    _email_fields = ('email_from', 'email_subject','email_body')
```
Comment (mvidalgarcia) [inline indico/modules/events/registration/forms.py]: You're missing a space after the comma.

**Comment by meluru** [comment]
Updated

File: indico/modules/events/registration/templates/emails/invitation_default.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{% macro get_subject() %}Invitation to register{% endmacro %}
+
+{% macro get_html_body() %}
+        Dear {first_name},<br>
```
Comment (ThiefMaster) [inline indico/modules/events/registration/templates/emails/invitation_default.html]: too much indentation

**Comment by ThiefMaster** [comment]
For the commit messages we usually use imperative style, e.g. "Support custom subject .." or "Allow custom subject .." if there's enough space in that line (<=51 chars).

https://chris.beams.io/posts/git-commit/ is a good read, and Vim actually highlights this nicely (first line too long, second line used at all == red "error"-style highlighting)

**Comment by meluru** [comment]
Updated

**Pull request merged**